fix.go

```diff
@@ -0,0 +1,56 @@
+package goose
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func Fix(db *sql.DB, dir string) error {
```

Comment:
we talked in person, we want to remove the database logic and keep `fix` to rename files only